### ISSUE TITLE
Several projects that could be run before are now running incorrectly

### ISSUE THREAD

**Issue by misitebao** [docsifyjs/docsify#1297]
#### Steps to reproduce
Several projects that could be run before are now running incorrectly

#### What is current behaviour
Previously, it could run completely, but now it can't run. There is no error in the console, but the page display is abnormal. The left menu and top navigation are missing, and there is a layer of mask.As shown in the figure.
![image](https://user-images.githubusercontent.com/28185258/88024772-d3934180-cb65-11ea-9fbd-72e5ea0f8219.png)

![image](https://user-images.githubusercontent.com/28185258/88024809-e3ab2100-cb65-11ea-856e-9da91c9ac72b.png)

#### What is the expected behaviour
Can be accessed normally

#### Other relevant information

- [ ] Bug does still occur when all/other plugins are disabled?

- Your OS: 
- Node.js version: 14.5.0 12,13,0 10.15.0 
- npm/yarn version: 6.14.5
- Browser version: chrome edge
- Docsify version: 4.4.1
- Docsify plugins: none

#### Please create a reproducible sandbox 

[![Edit 307qqv236](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/307qqv236)

**Comment by anikethsaha** [comment]
Please mention the version you are using and the version where it was working

**Comment by sy-records** [comment]
Please lower the version first, do not use the latest version. you can use `v4.11.3`.

```html
<!-- load css -->
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4.11.3/themes/vue.css">

<!-- load script -->
<script src="//cdn.jsdelivr.net/npm/docsify@4.11.3/lib/docsify.min.js"></script>
```

fixed via #1234

**Comment by misitebao** [comment]
Thank you for your reply. It's OK to switch to v4.11.3. The version number was not marked before, so it points to the master by default, that is, the previous version. Now that the warehouse is upgraded, the default master points to v4.11.4, so an error is reported. Thank you.
